src/main/scala/nl.knaw.dans.easy.validatebag/rules/ProfileVersion0.scala

```diff
@@ -37,7 +37,7 @@ object ProfileVersion0 {
     NumberedRule("1.1.1(datadir)", containsDir(Paths.get("data"))),
 
     // bag-info.txt
-    NumberedRule("1.2.1", containsFile(Paths.get("bag-info.txt"))),
+    NumberedRule("1.2.1", bagInfoExistsAndIsWellFormed),
```

Comment:
given the `AND` in your function name, shouldn't you just create two functions instead and use them in two separate `NumberedRule` instances? As done below, you could then call them `1.2.1(a)` and `1.2.1(b)`.

related to the implementation of this method: `containsFile` already has the correct shape to be put in a `NumberedRule`.